kotlinx-coroutines-core/common/src/Job.kt

```diff
@@ -404,6 +401,7 @@ public fun interface DisposableHandle {
  */
 @InternalCoroutinesApi
 @Deprecated(level = DeprecationLevel.ERROR, message = "This is internal API and may be removed in the future releases")
+@OptIn(InternalForInheritanceCoroutinesApi::class)
```

Comment:
just curious: why not to propagate optIn here? Overall, it's still has a lot of annotations, which screams "don't use this", but still (same for other `Job` APIs like `ParentJob` and `JobSupport`)

Reply:
To use a `Deprecated(ERROR)` API, you must suppress an error, which already proves your dedication and bravery beyond any doubt.